docs/sources/lids/template.md

```diff
@@ -0,0 +1,51 @@
+# Title
+
+> _NOTE: the file should be named `_<your-title>.md` and be placed in the `docs/sources/lids` directory. 
+Once accepted, it will be assigned a LID number and the file will be renamed by the sponsor.<br>
+> **Please remove this section before submitting, and retain the structure of this document.**_
+
+**Author:** Name (email)
+
+**Date:** MM/YYYY
+
+**Sponsor(s):** @username of maintainer(s) willing to shepherd this LID
+
+**Type:** Feature|Process
+
+**Status:** Draft|In Discussion|Accepted|Rejected|Completed
+
+**Related issues/PRs:**
+
+**Thread from [`#loki-dev` channel on Slack](https://slack.grafana.com):**
+
+---
+
+## Background
+
+_Describe the key facts and link to any documents, architecture diagrams or code. Keep this short, just a paragraph or two._
+
+## Problem Statement
+
+_Describe why the above is a problem we have now or will have in the future. Just a paragraph or two._
+
+## Goals
+
+_Describe the goals that we wish to achieve when considering the options available. Be clear about what is important and what could be left to the future. Non-goals are also useful to list here to clarify intent._
+
+## Proposals
+
+_List at least two possible ways of achieving the goals, these are typically different technical approaches that have differing trade-offs so summarise the trade-offs. “Do nothing” is typically worth considering too._
+
+### Proposal 0: Do nothing
+
+_Describe what happens if we do nothing, how bad does the problem become (over what time frame)?_
+
+### Proposal 1: Title
+
+_Describe the first proposal, what are the benefits and trade-offs that this approach has?_
+
+### Proposal n: Title
+
+_Describe the nth proposal(s), what are the benefits and trade-offs that these approaches have?_
```

Comment:
For the novice contributor I suggest here some optional subsections to make them frame and organize the proposal content easier. In Loki Operator as well as in upstream OpenShift's enhancement proposal processes we have sections like `Implemntation details`, `Risks and Mitigations`, `Design details`, etc.